modules/installation-and-upgrade/pages/proxy-container-installation.adoc

```diff
@@ -84,28 +84,28 @@ For more information about registering Salt client to the {productname} Server,
 
 {productname} Proxy containers are using [literal]``podman`` and [literal]``systemd`` to run and manage all proxy containers.
 
-First step is to install container control files provided by package [literal]``uyuni-proxy-systemd-services``.
+First step is to install the [literal]``mgrpxy`` utility provided by the package of the same name.
 
 
-[[proc-installation-proxy-containers-services]]
-.Procedure: Installation of container services for {productname} Proxy
+[[proc-installation-proxy-containers-mgrpxy]]
+.Procedure: Installation of container utility tool [literal]``mgrpxy`` for {productname} Proxy
 
 . Assign [literal]``Containers Module`` software channel to the container host in the {productname}.
 For more information about assigning software channels to the system, see xref:administration:channel-management.adoc[].
 
 . Log in as `root` on the container host.
 
-. Manually install {productname} Proxy service package:
+. Manually install [literal]``mgrpxy`` package:
 +
 
 ----
-zypper install uyuni-proxy-systemd-services
```

Comment:
As of today, I see this package present in the Manager tools for SLE Micro 5.5 product.

<img width="1093" alt="image" src="https://github.com/uyuni-project/uyuni-docs/assets/2827771/ebba47e0-3743-4a08-8a94-1e72886ee0bc">

But I can't see listed the package `mgrpxy`

Reply:
This is work in progress indeed

Reply:
PR merged now, should be done